### DISCUSSION TITLE
How to fetch rows of future date?

### DISCUSSION THREAD

**Discussion by MohammedHafiz** [supabase/supabase#9205] [Questions]
I have a table of orders and one of the columns is the date in string format and DD-MM-YYYY format. I'm currently fetching my queries like
const { data: bookings, error } = await supabase
          .from('bookings')
          .select("*")
          .gte('date', "28-09-2022')
this approach only works for the current month and not for the next month. For example, I'll get 30-09-2022 but not 01-10-2022.
Need help on this, thank you.

**Comment by saltcod** [comment]
Hey @MohammedHafiz — do you want a range instead maybe?

Some details in this discussion:
https://github.com/supabase/supabase/discussions/4812
